controllers/nginx/rootfs/Dockerfile

```diff
@@ -23,4 +23,10 @@ RUN DEBIAN_FRONTEND=noninteractive apt-get update && apt-get install -y \
 
 COPY . /
 
+# https://blog.phusion.nl/2015/01/20/docker-and-the-pid-1-zombie-reaping-problem
```

Comment:
can we just rely on the shell to manage zombies? 
eg http://blog.dscpl.com.au/2015/12/issues-with-running-as-pid-1-in-docker.html